core/src/data/value/error.rs

```diff
@@ -72,6 +72,13 @@ pub enum ValueError {
     #[error("unreachable failure on parsing number")]
     UnreachableNumberParsing,
 
+    #[error("failure to convert {a:#?} to {b:#?} for value:{value:#?}")]
+    ConversionErrorFromDataTypeAToDataTypeB {
+        a: DataType,
+        b: DataType,
+        value: Value,
+    },
```

Comment:
This looks for the other task. Current PR does not use `ConversionErrorFromDataTypeAToDataTypeB`

Reply:
Thanks for catching this..